src/emails/templates/breachAlert/BreachAlertEmail.tsx

```diff
@@ -73,17 +75,6 @@ export type RedesignedBreachAlertEmailProps = {
   breachedEmail: string;
   utmCampaignId: string;
   subscriber: SubscriberRow;
-  /**
-   * We need to run a bunch of queries to collect this data,
-   * so it's optional; however, make sure to pass it in for
-   * free users who are eligible for Plus (i.e. in the US),
-   * who have run a scan — those are the ones we show a
-   * <DataPointCount> for at the moment. We also use the
-   * presence of this property to determine whether free
-   * US users have run a scan. If, in the future, we assume
-   * we can do the same for Plus users, that might break, but
-   * that's a risk we'll have to take for now:
-   */
```

Comment:
Question: Is this comment not valid anymore?

Reply:
This is how it's done today, if you want to remove ambiguity it would be nice to have a function e.g. `hasRunScan()`, the implementation could just be to check that `onerep_profile_id` is set but that leaves us room to change in the future.